src/components/SettingsInputText/SettingsInputText.vue

```diff
@@ -0,0 +1,200 @@
+<!--
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
+  - @author Greta Doci <gretadoci@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<form ref="form"
+		:disabled="disabled"
+		@submit.prevent="onSubmit">
+		<div class="input-wrapper">
+			<label :for="id" class="action-input__label">{{ label }}</label>
+			<input :id="id"
+				type="text"
+				:value="value"
+				:disabled="disabled"
+				@input="onInput"
+				@change="onChange">
+			<input :id="idSubmit" type="submit" class="action-input__submit">
```

Comment:
If there is no value, the browser will add a fallback. We should propably control the text with our translation so that we use Nextcloud's user language and not the browsers :)

Something like ?
```suggestion
			<input :id="idSubmit" :value="t('Submit')" type="submit" class="action-input__submit">
```